src/SummaryStats.jl

```diff
@@ -0,0 +1,71 @@
+"""
+    SummaryStats
+
+Supertype for every type of summary statistic. 
+
+"""
+
+abstract type SummaryStats end
+
+"""
+    Calculate confidence intervals for given estimates.
+
+"""
+
+function confint(estimate::Float64, std_dev::Float64; type::String="normal", alpha::Float64=0.05)
```

Comment:
EDIT: After reviewing I think you can just copy the entire of `_ci` from 190 and rename it here.
- I think `estimate` and `std_dev` should be type `AbstractVector` (like in `_ci` in #190 ). This way you can pass whole column of estimates at the same time.
- I would keep `confint` in a different file `confint.jl`.